core/src/components/modal/modal.tsx

```diff
@@ -756,7 +771,7 @@ export class Modal implements ComponentInterface, OverlayInterface {
           part="content"
           ref={el => this.wrapperEl = el}
         >
-          {showHandle && <div class="modal-handle" part="handle"></div>}
+          {showHandle && <button class="modal-handle" part="handle" type="button" onClick={this.onHandleClick}></button>}
```

Comment:
Comparing with native iOS, VoiceOver will announce something like "sheet grabber, button, double tap to expand the sheet". When the sheet is expanded it will say "sheet grabber, button, double tap to collapse the sheet".

Ours just says "web dialog, button" which is not very helpful. We may not be able to get full parity with native, but can we find a way to make the read out more descriptive?